### PR TITLE
[ID] Add "osu!coin"

### PR DESCRIPTION
**Pull request by wowcake**
✈️

(https://osu.ppy.sh/wiki/en/osu%21coin)

### CODE REVIEW

File: wiki/osu!coin/id.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+stub: true
+tags:
+  - april fools
+---
+
+# osu!coin
+
+![osu!coin yang banyak](img/coins2x.png)![satu osu!coin](img/coin2x.png)
+
+**osu!coin** adalah sebuah lelucon, mata uang dalam permainan untuk mengejek taktik monetisasi pada permainan free-to-play lainnya. osu!coin diperkenalkan pada April Fools 2015, pemain harus membayar satu osu!coin agar bisa memainkan sebuah (satu) [beatmap](/wiki/Beatmap).
```
Comment (mousesalt) [inline wiki/osu!coin/id.md:11]:
```suggestion
**osu!coin** adalah sebuah lelucon mata uang dalam permainan untuk mengejek taktik monetisasi pada permainan free-to-play lainnya. osu!coin diperkenalkan pada April Fools 2015 yang mana diharuskan pemain untuk membayar satu osu!coin agar bisa memainkan sebuah (satu) [beatmap](/wiki/Beatmap).
```

File: wiki/osu!coin/id.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Rilis
+
+[Pernyataan resmi tentang osu!coins](https://osu.ppy.sh/home/news/2015-03-31-osucoins) menjelaskan bahwa osu!coin ditambahkan kepada permainan, sebagai cara untuk membiayai pesawat jet pribadi milik ![][flag_AU] [peppy](https://osu.ppy.sh/users/2), karena startegi monetisasi yang digunakan pada saat ini tidak akan bisa membiayai pesawat jet pribadi miliknya dalam masa hidupnya. Pernyataan tersebut berlanjut, menjelaskan bahwa setiap pemain baru akan diberikan 10 osu!coin, dan akan diberikan 10 lagi setiap harinya. Pemain lama akan diberikan dua tambahan osu!coin sebagai bonus. Pemain dapat mendapatkan lebih banyak osu!coin dengan mencapai kombo tertentu<!--TODO: link to combo multiplier-->. Satu osu!coin akan diberikan untuk setiap 100x kombo.
```
Comment (mousesalt) [inline wiki/osu!coin/id.md:15]:
```suggestion
[Pernyataan resmi tentang osu!coins](https://osu.ppy.sh/home/news/2015-03-31-osucoins) menjelaskan bahwa osu!coin ditambahkan ke dalam permainan, sebagai cara untuk membiayai pesawat jet pribadi milik ![][flag_AU] [peppy](https://osu.ppy.sh/users/2), karena strategi monetisasi yang digunakan pada saat ini tidak akan bisa membiayai pesawat jet pribadi miliknya dalam masa hidupnya. Pernyataan tersebut berlanjut, menjelaskan bahwa setiap pemain baru akan diberikan 10 osu!coin, dan akan diberikan 10 lagi setiap harinya. Pemain lama akan diberikan dua tambahan osu!coin sebagai bonus. Pemain dapat mendapatkan lebih banyak osu!coin dengan mencapai kombo tertentu<!--TODO: link to combo multiplier-->. Satu osu!coin akan diberikan untuk setiap 100x kombo.
```

File: wiki/osu!coin/id.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+Saat pemain kehabisan osu!coin, mereka tidak akan bisa memainkan beatmap apapun lagi. Tetapi, mereka dapat membeli lebih banyak - atas kemauan mereka sendiri - dalam harga tersebut:
```
Comment (mousesalt) [inline wiki/osu!coin/id.md:17]:
```suggestion
Saat pemain kehabisan osu!coin, mereka tidak akan bisa memainkan beatmap apapun lagi. Tetapi, mereka dapat membeli lebih banyak osu!coin - atas kemauan mereka sendiri - dengan harga berikut:
```

File: wiki/osu!coin/id.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+- $1 USD = 1 osu!coin
+- $10 USD = 10 osu!coin
+- $999 USD = 1000 osu!coin
+- anak pertama yang masih belum lahir = osu!coin yang tidak terbatas (anak yang belum lahir tersebut akan bekerja untuk osu!)
+
+Keesokan harinya, peppy membatalkan implementasi osu!coin, membuka subdomain yang baru untuk desain ulang osu!web dan diakhiri dengan [sebuah komentar mengenai perbaikan bug dan beberapa peningkatan kecil](https://osu.ppy.sh/comments/121803).
+
+## Penyambutan
+
+Meskipun implementasi untuk osu!coins menambahkan beberapa tambahan gambar grafik, osu!coin tidak memengaruhi pengalaman selagi pada bermain, pemain tetap bisa bermain meskipun batas osu!coin sudah habis. Sebaliknya, tidak ada cara yang tersedia untuk membeli osu!coin (meskipun pada option ada terdaftar).
```
Comment (mousesalt) [inline wiki/osu!coin/id.md:28]:
```suggestion
Meskipun implementasi untuk osu!coins menambahkan beberapa tambahan gambar grafik, osu!coin tidak memengaruhi pengalaman selagi pada bermain, pemain tetap bisa bermain meskipun batas osu!coin sudah habis. Sebaliknya, tidak ada cara yang tersedia untuk membeli osu!coin (meskipun pada menu opsi ada terdaftar).
```

**Review by mousesalt — APPROVED** [review]
should be fine now!

**Review by TicClick — APPROVED** [review]

**Pull request merged**